api/server.js

```diff
@@ -6,13 +8,39 @@ const authenticate = require('../auth/authenticate-middleware.js');
 const authRouter = require('../auth/auth-router.js');
 const jokesRouter = require('../jokes/jokes-router.js');
 
+const sessionOptions = {
+    name: 'mycookie',
+    secret: 'thisismycookie',
+    cookie: {
+      maxAge: 1000 * 60 * 60,
+      secure: false,
+      httpOnly: true, 
+    },
+    resave: false,
+    saveUninitalized: false,
```

Comment:
this was throwing an error for me on my machine. I think it's just because it's spelled wrong. You're missing an `i`